### PR TITLE
Option to group namespace specifiers

### LINKED ISSUE #104: Feature: Option to sort 'import * as lib from "lib"' first
It would be great if there was an option to sort `import * as lib from "lib"` at the top of the file.

For example

```javascript
import { LocationProvider } from "a-example-lib";
import * as bExampleLib from "b-example-lib";
import { RestaurantsProvider } from "c-example-lib";
```

to

```javascript
import * as bExampleLib from "b-example-lib";
import { LocationProvider } from "a-example-lib";
import { RestaurantsProvider } from "c-example-lib";
```

I tend to see these imports as globals or imports with the highest priority and I believe it would go a long way in keeping imports looking tidy

### PR DESCRIPTION
**Pull request by Mattinton**
This PR adds the `importOrderGroupNamespaceSpecifiers` config variable. When true it groups `import * as lib from 'lib'` imports to the top of the group.

Closes #104

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -32,6 +32,12 @@ const options = {
         default: false,
         description: 'Should imports be separated by new line?',
     },
+    importOrderGroupNamespaceSpecifiers: {
+        type: 'boolean',
+        category: 'Global',
+        default: false,
+        description: 'Should namespace specifies be grouped at the top?',
```
Comment (ayusharma) [inline src/index.ts:39]: I would like to have this feature. What do you think about a different approach here? May be we can do it by exposing a special keyword similar to [`THIRD_PARTY_MODULES` ](https://github.com/trivago/prettier-plugin-sort-imports#importorder). There can be cases where some people would like to keep them at a different locations in the import order.

Comment (Mattinton) [reply src/index.ts]: I agree, I would also occasionally like to group all namespace specifiers at the top in their own group, but I also like this current solution which would order the namespace specifiers at the top of each group. Is there a solution to suppport both?

Comment (ayusharma) [reply src/index.ts]: >I would also occasionally like to group all namespace specifiers at the top in their own group, but I also like this current solution which would order the namespace specifiers at the top of each group. 


I think I misunderstood your PR here. That is fine.  Could you please add this in Readme. md.

Comment (Mattinton) [reply src/index.ts]: Yes sorry, I think I had mislabelled my description. I've Added it to the README.

So this PR adds the ability to sort namespace specifiers to the top of the import group.

**Comment by ayusharma** [comment]
Hi @Mattinton, The build is failing. Could you please check it.

**Comment by Mattinton** [comment]
> Hi @Mattinton, The build is failing. Could you please check it.

Sorry should have checked this before the PR. Should be passing now.

**Comment by Mattinton** [comment]
Is there a standard release schedule for PR's like this?

**Comment by ayusharma** [comment]
> Is there a standard release schedule for PR's like this?

We will try to release the PR this week.

**Comment by ayusharma** [comment]
Hi @Mattinton , we had merged your changes to v3.2.x branch. We will try to release it as soon as possible along with some new changes. Thank you so much for your contribution ❤️

**Comment by Mattinton** [comment]
Hi, is there any progress on a release for this?

Cheers!

**Comment by Mattinton** [comment]
Sent again, let me know if any issues :D

**Comment by ayusharma** [comment]
Hi @Mattinton, I still have not received the mail. I checked the mail systems and they are working fine. Could you please drop the information to ayush.sharma@trivago.com and please cc behrang.yarahmadi@trivago.com

**Comment by Mattinton** [comment]
Sent again again 😆. Hoping it goes through this time!

**Comment by ayusharma** [comment]
I received it. Thank you.

**Pull request merged**